### ISSUE TITLE
Support for Vite.js

### ISSUE THREAD

**Issue by philippkuehn** [yjs/y-prosemirror#40]
**Describe the bug**
When using the next generation esm bundler [vite](https://github.com/vitejs/vite), I'm getting this error:

```bash
 > node_modules/y-prosemirror/src/lib.js:1:26: error: No matching export for import "ProsemirrorMapping"
    1 │ import { updateYFragment, ProsemirrorMapping } from './plugins/sync-plugin.js' // eslint-disable-line
      ╵                           ~~~~~~~~~~~~~~~~~~
```

I think this is because `ProsemirrorMapping` is a type generated in the build process. Since `vite` (esbuild) is very strict and uses the source files, it can't find this export. Here is a [comment from evan you](https://github.com/vitejs/vite/issues/1652#issuecomment-768560966) about that.

Do you have any idea how to fix that on your side?

**To Reproduce**
1. import `y-prosemirror` in a vite project
2. start dev server

**Expected behavior**
It should not throw an error

**Comment by holtwick** [comment]
I experienced the same problem. A fix would be great, thanks.

**Comment by holtwick** [comment]
I provided a PR that fixes the issue.

**Comment by dmonad** [comment]
Fixed this in y-prosemirror@1.0.7

**Comment by holtwick** [comment]
Thanks @dmonad Your patch works.
